src/TodoList.jsx

```diff
@@ -0,0 +1,33 @@
+let todoList = [
+    {
+        "id": "1",
+        "title": "Finish React Assignment"
+    },
+    {
+        "id": "2",
+        "title": "Submit React Assignment"
+    },
+    {
+        "id": "3",
+        "title": "Finish Node Assignment"
+    },
+    {
+        "id": "4",
+        "title": "Submit Node Assignment"
+    },
+]
+
+const TodoList = () => {
+    return (
+        <div>
+            <ul>
+                {todoList.map(
+                    (todo) => <li key={todo.id}>{todo.title}</li>
+                )
+                }
+            </ul>
+        </div>
+    )
+}
```

Comment:
Well done Chris!
Pro tip: Avoid using unnecessary elements to avoid overcomplicating your DOM structure. Since the `div` element in this component serves no purpose, feel free to remove it.

Reply:
Sorry, this took a while to address!  I appreciate the tip and is now addressed!